modules/liveIntentIdSystem.js

```diff
@@ -236,7 +237,9 @@ export const liveIntentIdSubmodule = {
       }
 
       if (value.thetradedesk) {
-        result.thetradedesk = { 'id': value.thetradedesk, ext: { provider: getRefererInfo().domain || LI_PROVIDER_DOMAIN } }
+        result.lipb = {...result.lipb, tdid: value.thetradedesk}
+        result.tdid = { 'id': value.thetradedesk, ext: { rtiPartner: 'TDID', provider: getRefererInfo().domain || LI_PROVIDER_DOMAIN } }
```

Comment:
Why `getRefererInfo().domain` here? Seems like we are not doing this elsewhere

Reply:
yes it was requested by Deepthi, https://github.com/LiveIntent/Prebid.js/pull/31#discussion_r1430514336